src/data/markdown/docs/02 javascript api/07 k6-experimental/04 tracing/02 Client.md

```diff
@@ -0,0 +1,95 @@
+---
+title: 'Client'
+excerpt: 'Client is a HTTP client attaching tracing information to its requests.'
+---
+
+`Client` is an HTTP client class that attaches tracing information to its requests. Use it to include a tracing context in HTTP requests so that tracing backends (such as [Grafana Tempo](https://grafana.com/oss/tempo/)) can incorporate their results.
```

Comment:
more nitpicking: I would prefer to (I think I did have this discussion with someone else) not use the word *class* as it has specific meanings for people coming from non JS and in js it has specific meanings and arguably we do not have classes *at all* within k6, for now. 

There isn't really a go API to create classes - we only have pure constructors which is what kt/experimental/tracing uses.

So I would recommend using `constructor`.

Reply:
Really good point indeed 👍🏻